config/src/main/resources/shandong.json

```diff
@@ -0,0 +1,870 @@
+{
+  "config": {
+    "chainId": 1337902,
+    "londonBlock": 0,
```

Comment:
Due to needing basefee in the genesis we add londonblock zero to trigger the logic.  In a future PR we should revisit this.

Reply:
We do extend the [parisDefinition](https://github.com/hyperledger/besu/blob/c4c0846976baae6b5fac9637a35e4f9b817bdec3/ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/MainnetProtocolSpecs.java#L642).

It's [this check](https://github.com/hyperledger/besu/blob/c4c0846976baae6b5fac9637a35e4f9b817bdec3/config/src/main/java/org/hyperledger/besu/config/GenesisConfigFile.java#L147) that requires the setting of any value for LondonBlock. I'll fix it in a separate PR.